src/components/Hello.vue

```diff
@@ -0,0 +1,16 @@
+<template>
```

Comment:
This is the "payoff" component. It's what we get to see if we pass authentication